tests/widget/operatorWithValidator/bondRewards/addBond.spec.ts

```diff
@@ -0,0 +1,207 @@
+/* eslint-disable no-irregular-whitespace */
+import { expect } from '@playwright/test';
+import { test } from '../../test.fixture';
+import { qase } from 'playwright-qase-reporter/playwright';
+import { Tags } from 'tests/consts/common.const';
+import { TOKEN_DISPLAY_NAMES } from 'utils/getTokenDisplayName';
+import { TOKENS } from 'consts/tokens';
+
+test.describe('Bond & Rewards. Add bond.', async () => {
+  test.beforeEach(async ({ widgetService }) => {
+    await widgetService.bondRewardsPage.addBond.open();
+  });
+
+  test(
+    qase(189, 'Should displays balance and explanatory text for stETH bond'),
+    async ({ widgetService, contractClients }) => {
+      const bondRewardsPage = widgetService.bondRewardsPage;
+
+      await test.step('Verify information about bond', async () => {
+        const nodeOperatorId = await widgetService.extractNodeOperatorId();
+
+        if (!nodeOperatorId) {
+          throw new Error('Node operator ID not found');
```

Comment:
wdyt to move throw error inside `widgetService.extractNodeOperatorId();`smiplify test context?